src/ontodia/viewUtils/connectionsMenu.tsx

```diff
@@ -786,12 +791,29 @@ class ObjectsPanel extends React.Component<ObjectsPanelProps, {
         );
     };
 
+    private counter = (activeObjCount: number) => {
+        const countString = `${activeObjCount}\u00A0of\u00A0${this.props.data.objects.length}`;
+
+        const wrongNodes =
+            Math.min(MAX_LINK_COUNT, this.props.data.linkDataChunk.expectedCount) - this.props.data.objects.length;
+        const wrongNodesString = Math.abs(wrongNodes) > MAX_LINK_COUNT ?
+            `${MAX_LINK_COUNT}+` : Math.abs(wrongNodes).toString();
+        const wrongNodesCount = wrongNodes === 0 ? '' : (wrongNodes < 0 ?
+            `\u00A0(${wrongNodesString})` : `\u00A0(${wrongNodesString})`);
+        const wrongNodesTitle = wrongNodes === 0 ? '' : (wrongNodes > 0 ? 'Unavailable nodes' : 'Extra nodes');
+
+        return <label className='ontodia-label ontodia-connections-menu_objects-panel_bottom-panel__count-label'>
```

Comment:
Please don't ever use `<label>` for non-form elements. It should be fine to use `<span>` instead.

Could you also rename `.ontodia-connections-menu_objects-panel_bottom-panel__count-label` class to something sane, like `.ontodia-connections-menu__items-count`?